c7n/filters/core.py

```diff
@@ -1018,3 +1020,123 @@ def reorder(self, items, key=None):
             return items[::-1]
         else:
             return sorted(items, key=key, reverse=(self.order == 'desc'))
+
+
+class FakeModel:
+    id = 'c7n:_id'
+
+
+class FakeFilterRegistry(FilterRegistry):
+
+    def __init__(self, *args, **kw):
+        super(FilterRegistry, self).__init__(*args, **kw)
+        self.register('value', ValueFilter)
```

Comment:
its not clear why this is being overridden, given this is all part of the super implementation.

Reply:
Yea i thought this wouldn't be needed but when I got rid of it the filter registration doesn't seem to work, and I'm not sure why that's happening